recipes/sphinxcontrib-redoc/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "sphinxcontrib-redoc" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c4fc2b432e08acdffe218b8e523b780eefd572d88de86f0a4afed4afec6e4087"
+
+build:
+  noarch: python 
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - setuptools_scm
+    - pip
+    - python
+
+  run:
+    - jinja2 >=2.4
+    - python
+    - six >=1.5
+    - sphinx >=1.5
+
+test:
+  imports:
+    - sphinxcontrib
```

Comment:
This doesn't actually test redoc importing at all

Reply:
@scopatz That should do it